testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -5774,6 +5774,12 @@ private static Matcher<SqlNode> isCharLiteral(String s) {
         .ok("(ARRAY[(ROW(1, 'a')), (ROW(2, 'b'))])");
   }
 
+  @Test void testArrayValueConstructorSpark() {
```

Comment:
In my opinion, RelToSqlConverterTest maybe better than SqlPraserTest. 
(Just a tip for you)

Reply:
+1 for removing the test to RelToSqlConverterTest